catalyst/registry.py

```diff
@@ -14,6 +14,15 @@ def _transforms_loader(r: registry.Registry):
 
     r.add_from_module(t, prefix=["catalyst.", "C."])
 
+    if SETTINGS.albu_required:
+        import albumentations as m
+
+        r.add_from_module(m, prefix=["A.", "albu.", "albumentations."])
+
+        from albumentations import pytorch as p
```

Comment:
for example, starting which version do we have PyTorch integrations under Albumentations?
and is it possible to wrap  
```python
from albumentations import pytorch as p
r.add_from_module(p, prefix=["A.", "albu.", "albumentations."])
```
with try-except? for the API change case? if there would be no `pytorch` integration?

Reply:
If I add a version of albumentations that supports pytorch integration, do I still need to wrap it via try-except?

Reply:
what if... the will break the API in a future releases? 🤔